### DISCUSSION TITLE
Adding the link to our work the repository details

### DISCUSSION THREAD

**Discussion by shawnthompson** [w3c/wcag2ict#73] [Ideas]
I suggest adding a link to the working or editor's draft in the details for this repository. 

You should be able to see a gear icon to the right of the About heading in the right panel.

![image](https://user-images.githubusercontent.com/2575266/206464953-bfb8563a-1bb3-47c4-8035-57701795fba1.png)

![image](https://user-images.githubusercontent.com/2575266/206464344-c39396d1-b914-4046-9815-c59ea5d1677d.png)

Alt: Edit repository details options

Doing this will add it to the panel on the right that list other important information for our work.

**Comment by maryjom** [comment]
@shawnthompson Sorry, for some reason I missed this post by you. Unfortunately I don't see a gear icon next to the About. Perhaps this is because I don't personally own the repository. There is a link to the draft in the Readme. So it's a little bit of a search to get to it, but at least it's readily available. @daniel-montalvo do you or someone else have access to the settings for the repository to add a link to the editors' draft in the right panel "About" content?

**Comment by daniel-montalvo** [comment]
Thanks @shawnthompson  and @maryjom 

Should be done now.
